kivy/utils.py

```diff
@@ -110,17 +110,34 @@ def rgba(s, *args):
     raise Exception('Invalid value (not a string / list / tuple)')
 
 
-def get_color_from_hex(s):
+def get_color_from_hex(color):
```

Comment:
`color`--> `hexcolorstr` ?

Reply:
Yes. but is just a argument name change.
I see what you are mean. You want the argument name to be more descriptive, no? But I assume the argument name is irrelevant and is self descriptive base on the function used same as `get_hex_from_color(color)`.